CHANGELOG.md

```diff
@@ -1,29 +1,40 @@
-# Virtual Attributes Changelog
+# Change Log
 
-Doing our best at supporting [SemVer](http://semver.org/) with
-a nice looking [Changelog](http://keepachangelog.com).
+Versioning is now following active record versioning. And not supporting SemVer.
```

Comment:
The wording of this statement implies a point in time, however this is an ephemeral statement.  That is, "now" won't make sense 5 versions in the future.  We should have the point-in-time statement in the versioned part as I commented in https://github.com/ManageIQ/activerecord-virtual_attributes/pull/104/files#r797922371.  Then, here, we can say something like:

```suggestion
Versioning of this gem follows ActiveRecord versioning, and does not support SemVer.
```